examples/tutorial/run.sh

```diff
@@ -17,7 +17,7 @@
 
 build/run \
   resources/tutorial/algo.asciipb \
-  NNIG resources/tutorial/nnig_ngg.asciipb \
+  MFA resources/tutorial/nnig_ngg.asciipb \
```

Comment:
> (Why did you open a new PR rather than using the old one?)
> 
> I have a fairly educated guess on why you are getting an extra column from the CSV file. It may have to do with line endings. To make a long story short, both Windows and Unix systems (such as Linux) use different representations of newline characters (CRLF vs LF), and it's statistically likely that you are running Windows. All other working CSV files in this repository use Unix-style newline characters. I tried downloading your CSV file, and guess what, my text editor tells me it is using Windows-style newlines. There are several options to change from CRLF to LF, and sometimes text editors have built-in methods to perform this conversion. There also exist one-line commands which you can run from Bash terminals such as Git Bash for Windows. Just look up "crlf to lf <YOUR EDITOR NAME>" or "crlf to lf bash", and you'll find what you need.
> 
> Some other code-related comments follow :)

Yes! I think you are right because when I created a new csv file directly in Linux it no longer needs to be resized.